core/templates/dev/head/domain/exploration/ExplorationObjectFactory.js

```diff
@@ -132,6 +132,22 @@ oppia.factory('ExplorationObjectFactory', [
         languageCode);
     };
 
+    Exploration.prototype.getAllAudioTranslations = function(languageCode) {
+      return this.states.getAllAudioTranslations(languageCode);
+    };
+
+    Exploration.prototype.getAllAudioTranslationsFileSizeMB =
+      function(languageCode) {
+        var NUM_BYTES_IN_MB = 1 << 20;
+        var totalFileSizeBytes = 0;
+        var allAudioTranslations =
+          this.states.getAllAudioTranslations(languageCode);
+        allAudioTranslations.map(function(audioTranslation) {
+          totalFileSizeBytes += audioTranslation.fileSizeBytes;
```

Comment:
Optional: Might as well reuse audioTranslation.getFileSizeMB()? At the very least you don't then duplicate NUM_BYTES_IN_MB across the codebase.